apps/test/unit/templates/SignInOrAgeDialogTest.js

```diff
@@ -99,14 +107,15 @@ describe('SignInOrAgeDialog', () => {
         .find('Button')
         .at(1)
         .simulate('click');
-      assert.strictEqual(sessionStorage.getItem('anon_over13'), 'true');
+      assert.equal(setItemSpy.callCount, 1);
```

Comment:
Could make sure that the correct session storage value was set:
```suggestion
      assert(setItemSpy.calledOnce);
      assert(setItemSpy.calledWith('anon_over13', true));
```